src/cljs/witan/ui/activities.cljs

```diff
@@ -169,7 +169,13 @@
   (let [{:keys [activity] :as app-state}
         (data/get-in-app-state :app/activities :activities/pending id)]
     (when (and app-state (= activity-name activity))
-      (finish-activity! nil :failed id))))
+      ;; FIXME - Change message to something useful from a reporter.
+      (do
+        (data/publish-topic :activity/panic {:message "something"
```

Comment:
I'm in two minds about whether or not we need this as the `finish-activity!` does catch the error (but not as clearly as the panic will). I wonder if they are 2 different things.